src/main/java/seedu/address/logic/Logic.java

```diff
@@ -51,4 +52,9 @@ public interface Logic {
      * Set the user prefs' GUI settings.
      */
     void setGuiSettings(GuiSettings guiSettings);
+
+    /**
+     * Returns the data needed to generate the PieChart Statistics
```

Comment:
Perhaps we should explicitly state that its a list so its not misinterpreted as a single data point!
```suggestion
     * Returns an observable list of data needed to generate the PieChart Statistics
```

Reply:
or this one that u wrote for the same method in a different class works too:
Return the list of Statistical PieChart datapoints

Reply:
oh ok will use the one i wrote in a diff class thx!